Assets/Scripts/Capabilities/Dash.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Dash : MonoBehaviour
+{
+    [SerializeField] InputController input = null;
+    [SerializeField] MovementOverride movement;
+    [SerializeField] Rigidbody2D rb;
+    [SerializeField] PlayerMovementController controller;
+    
+    private bool isDashing = false;
+    private Vector3 dashDirection;
+    private float dashSpeed;
+
+
+
+    void Update()
+    {
+        if (input.RetrieveDashInput())
+        {
+            isDashing = true;
+            dashDirection = new Vector3(1,0,0);
```

Comment:
no change needed - fyi shortcut for this is `Vector3.right`

```suggestion
            dashDirection = Vector3.right;
```

Reply:
https://docs.unity3d.com/ScriptReference/Vector3-right.html

Reply:
Right, I forgot about that. I'll update it